app/config/translations.json

```diff
@@ -126,13 +141,19 @@
     "Fin Source": "Funds"
   },
   "es": {
```

Comment:
As far as I'm concerned, we can stop using the spreadsheet altogether.
We might consider using a crowdsources translation service (e.g. lokalise.co, crowdin.com, transifex.com etc.), but for now we can edit the json files.
In the npm build process we can do the combination of the different files into one big JSON file.